### PR TITLE
Theme/main page

### PR DESCRIPTION
**Pull request by YakovchukIvan**

### CODE REVIEW

File: src/components/Layout/Layout.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import Header from './header/index'
 import Breadcrumbs from './breadcrumbs/Breadcrumbs'
 import Footer from './footer/index'
 
-const Layout = () => {
+function Layout() {
```
Comment (pqoqubbw) [inline src/components/Layout/Layout.tsx]: why did you change type of function here?

File: src/components/hero-section/HeroSection.module.scss [hunk 2]
```diff
@@ -69,3 +69,7 @@
     background-color: $yellow-200;
   }
 }
+
+.test {
```
Comment (pqoqubbw) [inline src/components/hero-section/HeroSection.module.scss]: remove this

File: src/components/hero-section/heroSection.tsx [hunk 3]
```diff
@@ -1,55 +1,115 @@
...
+              height={205}
+              component="img"
+              src={HeroPic}
+              alt="dried fruits in plastic bags"
+              sx={{
+                ...sloganImgLogo.lg,
+                [theme.breakpoints.between('sm', 'md')]: sloganImgLogo.md
+              }}
+            />
+          )}
```
Comment (pqoqubbw) [inline src/components/hero-section/heroSection.tsx]: you can do it in ease way, something like this

```suggestion
          <Box
          height={isSmallScreen ? undefined : 205}
          component="img"
          src={isSmallScreen ? HeroMobilePic : HeroPic}
          alt="dried fruits in plastic bags"
          sx={isSmallScreen ? sloganImgLogo.sm : {...sloganImgLogo.lg, [theme.breakpoints.between('sm', 'md')]: sloganImgLogo.md}}
        />
```

File: src/components/section-about-us/style.ts
Comment (pqoqubbw) [inline src/components/section-about-us/style.ts:1]: if you reuse these styles - fine, for now as a separate style, in the future we will remove them too. if not, move the styles directly to the needed component

File: src/components/hero-section/style.ts
Comment (pqoqubbw) [inline src/components/hero-section/style.ts:1]: if you reuse these styles - fine, for now as a separate style, in the future we will remove them too. if not, move the styles directly to the needed component

File: src/components/section-faq/accordionData.ts [hunk 4]
```diff
@@ -0,0 +1,54 @@
+export const accordionData = [
```
Comment (pqoqubbw) [inline src/components/section-faq/accordionData.ts]:
```suggestion
export const ACCORDION_DATA = [
```

File: src/components/section-faq/style.ts
Comment (pqoqubbw) [inline src/components/section-faq/style.ts:1]: if you reuse these styles - fine, for now as a separate style, in the future we will remove them too. if not, move the styles directly to the needed component

File: src/components/section-reviews/sectionReviews.tsx [hunk 5]
```diff
@@ -2,12 +2,28 @@ import Slider, { Settings } from 'react-slick'
...
+import {
+  border,
+  borderSecond,
+  iconBwrapper,
+  iconWrapper,
+  sectionBg,
+  sliderContainer
+} from './style'
+
+const reviewsClients = [
```
Comment (pqoqubbw) [inline src/components/section-reviews/sectionReviews.tsx]:
```suggestion
const REVIEWS_CLIENTS = [
```

File: src/components/section-reviews/style.ts
Comment (pqoqubbw) [inline src/components/section-reviews/style.ts:1]: if you reuse these styles - fine, for now as a separate style, in the future we will remove them too. if not, move the styles directly to the needed component

File: src/components/section-slider/arrowNext.tsx [hunk 6]
```diff
@@ -1,16 +1,17 @@
...
 type Props = {
   onClick?: () => void
 }
 
 const ArrowNext: FC<Props> = ({ onClick }) => {
   return (
-    <div className={styles.ArrowNextStyle} onClick={onClick}>
-      <ArrowIcon className={styles.arrow} />
-    </div>
+    <Box className={`${styles.arrow} ${styles.next}`} onClick={onClick}>
```
Comment (pqoqubbw) [inline src/components/section-slider/arrowNext.tsx]: use `clsx`

File: src/components/section-slider/arrowPrev.tsx [hunk 7]
```diff
@@ -1,16 +1,17 @@
...
 type Props = {
   onClick?: () => void
 }
 
 const ArrowPrev: FC<Props> = ({ onClick }) => {
   return (
-    <div className={styles.ArrowPrevStyle} onClick={onClick}>
-      <ArowIcon className={styles.arrow} />
-    </div>
+    <Box className={`${styles.arrow} ${styles.prev}`} onClick={onClick}>
```
Comment (pqoqubbw) [inline src/components/section-slider/arrowPrev.tsx]: use `clsx`

File: src/components/section-slider/customSlider.tsx [hunk 8]
```diff
@@ -1,30 +1,98 @@
...
+  sliderButton,
+  slideInfo
+} from './style'
+import { Button } from '../UI/Button'
 
 type Props = HTMLProps<HTMLDivElement> & { index: number }
 
-const CustomSlider: FC<Props> = ({ ...props }) => {
+const CustomSlider: FC<Props> = (props) => {
+  const { index } = props
```
Comment (pqoqubbw) [inline src/components/section-slider/customSlider.tsx]:
```suggestion
const CustomSlider: FC<Props> = ({ index }) => {
```

File: src/components/section-your-choice/sectionYourChoice.tsx [hunk 9]
```diff
@@ -4,82 +4,110 @@ import IconBowl from '../../icons/bowl.svg?react'
...
+import {
+  paragraphItalic,
+  paragraphWrapper,
+  yourChoiceList,
+  yourChoiceListItem,
+  yourChoiceParagraph,
+  yourChoiceWrapper
+} from './style'
+
+const itemsList = [
```
Comment (pqoqubbw) [inline src/components/section-your-choice/sectionYourChoice.tsx]:
```suggestion
const ITEMS_LIST = [
```

File: src/components/section-your-choice/style.ts
Comment (pqoqubbw) [inline src/components/section-your-choice/style.ts:1]: if you reuse these styles - fine, for now as a separate style, in the future we will remove them too. if not, move the styles directly to the needed component

**Review by pqoqubbw — CHANGES_REQUESTED** [review]
also, please, do not forget to delete all commented data from components

File: src/components/section-about-us/sectionAboutUs.tsx [hunk 10]
```diff
@@ -1,38 +1,87 @@
...
+          sx={{
+            [theme.breakpoints.up('sm')]: { display: 'none' }
+          }}
+        />
+        <Box
+          component="img"
+          src={FruitMix}
+          alt="fruit mix"
+          width="100%"
+          // sx={{ [theme.breakpoints.up('md')]: { width: '50%' } }}
```
Comment (pqoqubbw) [inline src/components/section-about-us/sectionAboutUs.tsx]: remove unused code

File: src/components/section-about-us/sectionAboutUs.tsx [hunk 11]
```diff
@@ -1,38 +1,87 @@
...
-          </span>
+          <Typography
+            variant="caption"
+            className={styles.aboutUsParagraphItalic}
+          >
+            {' '}
+            яблука!{' '}
+          </Typography>
+        </Typography>
+        {/* <br /> */}
```
Comment (pqoqubbw) [inline src/components/section-about-us/sectionAboutUs.tsx]: remove unused code

File: src/components/section-reviews/sectionReviews.tsx [hunk 12]
```diff
@@ -1,93 +1,145 @@
...
-        ></div>
+        />
       )
     },
     beforeChange: (_, next) => {
       setActiveSlide(next)
     }
   }
 
+  const CreateBorderStyle = (url: string, positionWave: 'top' | 'bottom') => ({
```
Comment (pqoqubbw) [inline src/components/section-reviews/sectionReviews.tsx]:
```suggestion
  const createBorderStyle = (url: string, positionWave: 'top' | 'bottom') => ({
```

**Review by pqoqubbw — CHANGES_REQUESTED** [review]

File: src/components/hero-section/HeroSection.module.scss (hunk 2 above)
Comment (YakovchukIvan) [reply src/components/hero-section/HeroSection.module.scss]: Done

File: src/components/Layout/Layout.tsx (hunk 1 above)
Comment (YakovchukIvan) [reply src/components/Layout/Layout.tsx]: Done

File: src/components/hero-section/heroSection.tsx (hunk 3 above)
Comment (YakovchukIvan) [reply src/components/hero-section/heroSection.tsx]: Done

File: src/components/section-about-us/style.ts
Comment (YakovchukIvan) [reply src/components/section-about-us/style.ts:1]: Done

File: src/components/hero-section/style.ts
Comment (YakovchukIvan) [reply src/components/hero-section/style.ts:1]: Done

File: src/components/section-slider/arrowNext.tsx (hunk 6 above)
Comment (YakovchukIvan) [reply src/components/section-slider/arrowNext.tsx]: Done

File: src/components/section-slider/arrowPrev.tsx (hunk 7 above)
Comment (YakovchukIvan) [reply src/components/section-slider/arrowPrev.tsx]: Done

File: src/components/section-slider/customSlider.tsx (hunk 8 above)
Comment (YakovchukIvan) [reply src/components/section-slider/customSlider.tsx]: Done

File: src/components/section-your-choice/sectionYourChoice.tsx (hunk 9 above)
Comment (YakovchukIvan) [reply src/components/section-your-choice/sectionYourChoice.tsx]: Done

File: src/components/section-your-choice/style.ts
Comment (YakovchukIvan) [reply src/components/section-your-choice/style.ts:1]: Done

File: src/components/section-about-us/sectionAboutUs.tsx (hunk 10 above)
Comment (YakovchukIvan) [reply src/components/section-about-us/sectionAboutUs.tsx]: Done

File: src/components/section-about-us/sectionAboutUs.tsx (hunk 11 above)
Comment (YakovchukIvan) [reply src/components/section-about-us/sectionAboutUs.tsx]: Done

File: src/components/section-reviews/sectionReviews.tsx (hunk 12 above)
Comment (YakovchukIvan) [reply src/components/section-reviews/sectionReviews.tsx]: Done

**Review by pqoqubbw — APPROVED** [review]
